openpathsampling/storage/storage.py

```diff
@@ -25,6 +25,8 @@ class Storage(NetCDFPlus):
     simulation. This allows effective storage of shooting trajectories
     """
 
+    USE_FEATURE_SNAPSHOTS = False
```

Comment:
> I realized though that the new ToySnapshot seems to be faster, but have not tested it. In my feeling the MSTIS examples are running faster now. This is because all the energy handling and creating of Configuration objects is not necessary anymore.

Primarily energy calculation. Keep in mind that, with only 2 degrees of freedom, energy calculation is about half the cost of force calculation (and is not calculated as part of the force calculation). So you probably see ~66% time cost. This is why we don't usually do that with simple models; I added it because it was a part of the original snapshot.